### ISSUE TITLE
sentiment issues

### ISSUE THREAD

**Issue by jennybc** [ropenscilabs/available#58]
The "sentiment" part of my results is not getting populated.

I poked around a wee bit and also notice this tidytext workaround that can probably get dismantled now:

https://github.com/ropenscilabs/available/blob/2a54276f8c1ebe3c77f907fecf90d626b181730a/R/sentiment.R#L6-L9

Indeed, changes in tidytext probably explain why sentiment analysis isn't working right now.

**Comment by jennybc** [comment]
Hmm ... maybe it's just related to the package name(s) I was checking. The refactoring of this function is still a good idea though.

**Comment by jkylearmstrong** [comment]
@jennybc agree, expecting that the package name is going to be something in the dictionary is unlikely, so why not use cosine distance since we are looking for sentiment?
